librarymanagement/src/main/contraband/librarymanagement.json

```diff
@@ -138,19 +138,20 @@
           "namespace": "sbt.librarymanagement",
           "target": "Scala",
           "doc": [
-            "Cross-versions a module using the result of applying `remapVersion` to the binary version.",
-            "For example, if `remapVersion = v => \"2.10\"` and the binary version is \"2.9.2\" or \"2.10\",",
-            "the module is cross-versioned with \"2.10\"."
+            "Cross-versions a module using the result of appending `suffix` to the binary version.",
+            "For example, if `suffix = \"_foo\"` and the binary version is \"2.10\",",
+            "the module is cross-versioned with \"2.10_foo\"."
```

Comment:
Oh. In Scala.js and sbt-crossproject it's the other way around: `"foo_2.10"` (hence the full artifact name is `artifactname_foo_2.10`. This was not an arbitrary choice. We needed `_2.10` to remain the last part for tooling not to choke. For example, the "suspicious binary incompatibility" detector of the Scala IDE (which severely warns if you use a _2.10 dependency in a 2.11 project) requires the Scala binary version suffix to be last.